ci/container/test_component.sh

```diff
@@ -31,12 +31,12 @@ PACKAGE_NAME=$(cd $WORKSPACE && ls snowflake-sdk*.tgz)
 npm install $WORKSPACE/${PACKAGE_NAME}
 
 echo "[INFO] Setting test parameters"
-if [[ -f "$WORKSPACE/parameters.json" ]]; then
-    echo "[INFO] Found parameter file in $WORKSPACE"
-    PARAMETER_FILE=$WORKSPACE/parameters.json
-else
+if [[ "$LOCAL_USER_NAME" == "jenkins" ]]; then
     echo "[INFO] Use the default test parameters.json"
     PARAMETER_FILE=$SOURCE_ROOT/test/parameters.json
+else
+    echo "[INFO] Found parameter file in $WORKSPACE"
+    PARAMETER_FILE=$WORKSPACE/parameters.json
```

Comment:
We can remove this if  block . Anyway it needs to use test/parameters.json file right?

Reply:
The github tests still uses $WORKSPACE/parameters.json. It's only jenkins test that uses test/parameters.json